notifications/rest/bindings/list-binding/output/list-binding.json

```diff
@@ -1,14 +1,14 @@
 {
   "bindings": [
-        {
-      "endpoint": "endpoint_id",
+    {
+      "endpoint": "eyJjdHkiOiJ0d2lsaW8tZnBhO3Y9MSIsInR5cCI6IkpXVCIsImFsZyI6IkhTMjU2In0.eyJncmFudHMiOnsiY2hhdCI6eyJzZXJ2aWNlX3NpZCI6IklTNmIwZmE2MDgzOWM1MTFlNWExNTFmZWZmODE5Y2RjOWYiLCJlbmRwb2ludF9pZCI6InR3aTEtMTdjNTkyM2Y5NTBiNGJhYjllZjAwZTJlZjdkYTBkYTIifSwiaWRlbnRpdHkiOiJOb3RpZmljYXRpb25MaWJyYXJ5VW5pdFRlc3RlcjEifSwic3ViIjoiQUM3OGU4ZTY3ZmMwMjQ2NTIxNDkwZmI5OTA3ZmQwYzE2NSIsImlzcyI6IlNLMmQxYWRmOGIxYzY0YTYzM2U1ZjE4NjU5ZjdjZDE5MTEiLCJleHAiOjE0NzgyNTI4NTl9.1i256CwtacP859Xh-6cEZVio4oohxL-od9krzsa4DnY",
       "notification_protocol_version": "default",
       "tags": [
         "new user",
         "preferred device"
       ],
       "date_updated": "2017-01-17T14:26:42Z",
-      "address": "+44XXXXXXXXXX",
+      "address": "+441234123456",
```

Comment:
Again, it's better to not use a unique number in these snippets, unless every user's code needs exactly this address